x-pack/platform/test/rule_registry/security_and_spaces/tests/basic/tags.ts

```diff
@@ -0,0 +1,237 @@
+  /** Unauthorized users */
+  unauthorizedUsers: User[];
+}
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const esArchiver = getService('esArchiver');
+
+  const TEST_URL = '/internal/rac/alerts';
+  const ALERTS_INDEX_URL = `${TEST_URL}/index`;
+  const SPACE1 = 'space1';
+  const SPACE2 = 'space2';
+  const APM_ALERT_ID = 'NoxgpHkBqbdrfX07MqXV';
+  const APM_ALERT_INDEX = '.alerts-observability.apm.alerts';
+  const SECURITY_SOLUTION_ALERT_ID = '020202';
+  const SECURITY_SOLUTION_ALERT_INDEX = '.alerts-security.alerts';
+
+  const getAPMIndexName = async (user: User) => {
+    const { body: indexNames }: { body: { index_name: string[] | undefined } } =
+      await supertestWithoutAuth
+        .get(`${getSpaceUrlPrefix(SPACE1)}${ALERTS_INDEX_URL}`)
+        .auth(user.username, user.password)
+        .set('kbn-xsrf', 'true')
+        .expect(200);
+    const observabilityIndex = indexNames?.index_name?.find(
+      (indexName) => indexName === APM_ALERT_INDEX
+    );
+    expect(observabilityIndex).to.eql(APM_ALERT_INDEX); // assert this here so we can use constants in the dynamically-defined test cases below
+  };
+
+  const getSecuritySolutionIndexName = async (user: User) => {
+    const { body: indexNames }: { body: { index_name: string[] | undefined } } =
+      await supertestWithoutAuth
+        .get(`${getSpaceUrlPrefix(SPACE1)}${ALERTS_INDEX_URL}`)
+        .auth(user.username, user.password)
+        .set('kbn-xsrf', 'true')
+        .expect(200);
+    const securitySolution = indexNames?.index_name?.find((indexName) =>
+      indexName.startsWith(SECURITY_SOLUTION_ALERT_INDEX)
+    );
+    expect(securitySolution).to.eql(`${SECURITY_SOLUTION_ALERT_INDEX}-${SPACE1}`); // assert this here so we can use constants in the dynamically-defined test cases below
+  };
+
+  describe('Alert - Bulk Update tags - RBAC - spaces', () => {
+    before(async () => {
+      await getSecuritySolutionIndexName(superUser);
+      await getAPMIndexName(superUser);
+    });
+
+    before(async () => {
+      await esArchiver.load('x-pack/platform/test/fixtures/es_archives/rule_registry/alerts');
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/platform/test/fixtures/es_archives/rule_registry/alerts');
+    });
+
+    function addTests({ space, authorizedUsers, unauthorizedUsers, alertId, index }: TestCase) {
+      authorizedUsers.forEach(({ username, password }) => {
+        it(`${username} should bulk update alert tags with given id ${alertId} in ${space}/${index}`, async () => {
+          const { body: updated } = await supertestWithoutAuth
+            .patch(`${getSpaceUrlPrefix(space)}${TEST_URL}/tags`)
+            .auth(username, password)
+            .set('kbn-xsrf', 'true')
+            .send({
+              alertIds: [alertId],
+              addTags: ['new-tag'],
```

Comment:
I noticed all these only have the `addTags` param. Could you please add one scenario for tag removal? (No need to duplicate all tests, just one `should bulk remove alert tags`.)